Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/UiBackendApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
 @SpringBootApplication
+@EntityScan(value = {"it.deltax.produlytics.persistence"})
```

Comment:
Wait, sta roba è necessaria? 😂

Reply:
Eh sì purtroppo

Spring non è abbastanza intelligente da scannerizzare le entities di un altro package in autonomia

Se ci pensi ha anche senso cioè altrimenti dovrebbe scannerizzare TUTTI i package di TUTTI i pacchetti installati 😬

Reply:
Mi pareva ci fosse un metodo di ottenere tutte le classi con una certa annotazione dai preprocessor di java... Forse lo fa a runtime?